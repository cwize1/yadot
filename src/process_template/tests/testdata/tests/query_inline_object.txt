animals:
- cat: ${{ .cat }}
- dog: ${{ .dog }}
