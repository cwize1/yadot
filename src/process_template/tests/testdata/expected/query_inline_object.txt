ERROR: <None>
OUTPUT:
---
animals:
  - cat:
      motion: pounce
  - dog:
      motion: run